### DISCUSSION TITLE
Export SVG/CSS Animation as GIF

### DISCUSSION THREAD

**Discussion by SethFalco** [ImageMagick/ImageMagick#2391] [Help] [answered]
I'm currently trying to see if there's a good way export an CSS animation as a GIF.

I started with an HTML/CSS animation, but I'm currently in the process of making it an SVG/CSS animation instead.

I was just wondering, does ImageMagick have the capability to export an SVG animation (`.svg`) as an animated GIF?

**Comment by snibgo** [comment, marked as answer]
As far as I know, Inkscape can't rasterise individual frames from animated SVG. If Inkscape can't, then neither can ImageMagick.

But perhaps Inkscape can, now. Otherwise, what tool would you suggest?

**Reply by SethFalco** [reply]
I see; that's unfortunate.

I wish I could suggest something, but I was really betting on ImageMagick (or Inkscape) myself, for now, all I've done is a shoddy open in browser, screen capture, and remove background solution.
